internal/pkg/cli/deploy/svc.go

```diff
@@ -1360,6 +1360,9 @@ func (d *backendSvcDeployer) validateALBRuntime() error {
 
 func (d *lbWebSvcDeployer) validateALBRuntime() error {
 	hasImportedCerts := len(d.envConfig.HTTPConfig.Public.Certificates) != 0
+	if d.lbMft.RoutingRule.Redirect != nil && *d.lbMft.RoutingRule.Redirect && d.app.Domain == "" && !hasImportedCerts {
```

Comment:
What is the difference between 1st two conditions?

Reply:
`RoutingRule.Redirect` is `*bool`, so it's basically saying "if redirect is configured and set to true". If we didn't have the first condition then the second would panic if `Redirect` was not configured.